todo_app/data/decorators/user_role_decorator.py

```diff
@@ -0,0 +1,13 @@
+from functools import wraps
+from flask import abort
+
+import flask_login
+
+
+def requires_write_access(f):
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        if not flask_login.current_user.has_write_access():
```

Comment:
```suggestion
        if not flask_login.current_user.has_write_access:
```
Not sure if this converted from a function to a field at some point, but this errored for me when I ran it